ui/app/rif/pages/rns/register/index.js

```diff
@@ -258,43 +269,40 @@ class DomainRegisterScreen extends Component {
         </div>
       ),
     };
-    if (currentStep === 'registering' && this.props.registeringProgress <= 120) {
+    if (currentStep === 'waitingForRegister') {
```

Comment:
Maybe we can move this strings to constants, or its irrelevant?

Reply:
It's internal to the component, i mean it's for internal management of the component so i guess is not necessary